neqo-transport/src/path.rs

```diff
@@ -573,14 +578,37 @@ impl Path {
             challenge: None,
             rtt: RttEstimate::default(),
             sender,
-            tos: IpTos::default(), // TODO: Default to Ect0 when ECN is supported.
-            ttl: 64,               // This is the default TTL on many OSes.
+            tos: IpTosEcn::Ect0.into(),
```

Comment:
Do we need to make `ecn_enabled` configurable by a gecko pref?
I assume the risk of this should be low, but I am not super sure.

Reply:
Not sure. I assume for TCP-based networking, we are letting the OS decide what to do about ECN?

Reply:
The general sentiment from IETF folks doing this stuff is that ECN is safe to use.  I consider it low risk, particularly with the defense against drops and bleaching that is included in this patch.